src/libcommonserver/io/iohelper_win.cpp

```diff
@@ -334,7 +335,8 @@ bool IoHelper::checkIfFileIsDehydrated(const SyncPath &itemPath, bool &isDehydra
 }
 
 static bool setRightsWindowsApi(const SyncPath &path, DWORD permission, ACCESS_MODE accessMode, IoError &ioError,
-                                log4cplus::Logger logger) noexcept {  // Always return false if ioError != IoErrorSuccess, caller
+                                log4cplus::Logger logger,
+                                bool inherite = false) noexcept {             // Always return false if ioError != IoErrorSuccess, caller
```

Comment:
The comment is written in a strange way on 2 lines. Put it on the line above the definition since it is a description of the function behavior.

Reply:
CLang split the comment on two lines as it was too long. 
I moved it above the definition as suggested (commit: https://github.com/Infomaniak/desktop-kDrive/pull/115/commits/032300bac6dbd7e4fe32c6d68e7aa5176dc6a044), this solved the problem.